include/revng/EarlyFunctionAnalysis/BasicBlock.h

```diff
@@ -24,34 +24,30 @@ type: struct
       End address of the basic block, i.e., the address where the last
       instruction ends
     type: MetaAddress
-  - name: CustomName
```

Comment:
So, I guess we're getting rid of basic block names. I feel like there might be a subset of users who'd prefer being able to "mark" some of them for future reference, etc?

Maybe something like having a separate `sequence` in the model specifying "labels" for specific addresses (at that point, maybe we shouldn't even limit it to basic blocks only) worth considering in the future?

P. S. Going even crazier, it might be really cool to carry those "labels" on through the decompiler in some capacity and preserve them in like c-style `label:`s or even just comments - might make working with the decompiled code easier.

Reply:
Eh, you're right. We'd need something like `CallSitePrototypes` to specify these optional kind of information.
But I'd postpone this.